testApp/src/InitializeMethods.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { Didomi } from '@didomi/react-native';
+import MethodCall from './MethodCall';
+
+interface InitializeProps {
+  updateSdkState: (state: string) => any;
+}
+
+export default function InitializeMethods(props: InitializeProps) {
```

Comment:
I'd suggest adding a description here so it's clear what other methods should be added here.